misc/src/main/java/com/example/snippets/profiling/ProfilingManagerJavaSnippets.java

```diff
@@ -125,5 +128,85 @@ public void setupProfileUploadWorker(String resultFilePath) {
       // Setup job to upload the profiling result file.
     }
     // [END android_profiling_manager_triggered_trace_setup_upload_job_java]
+
+    // [START android_profiling_manager_anr_case_study_java_snippet_1]
+    public void addANRTrigger() {
+      ProfilingManager profilingManager = getApplicationContext().getSystemService(ProfilingManager.class);
+      List<ProfilingTrigger> triggers = new ArrayList<>();
+      ProfilingTrigger.Builder triggerBuilder = new ProfilingTrigger.Builder(ProfilingTrigger.TRIGGER_TYPE_ANR);
+      triggers.add(triggerBuilder.build());
+      Executor mainExecutor = Executors.newSingleThreadExecutor();
+      Consumer<ProfilingResult> resultCallback =
+          profilingResult -> {
+            // Handle uploading trace to your back-end
+          };
+      profilingManager.registerForAllProfilingResults(mainExecutor, resultCallback);
+      profilingManager.addProfilingTriggers(triggers);
+    }
+    // [END android_profiling_manager_anr_case_study_java_snippet_1]
+
+    // [START android_profiling_manager_anr_case_study_java_snippet_2]
```

Comment:
Not sure which section this is intended for, this doesn't seem like it would be related to ANR triggers? Looks like a manual trigger